src/vaadin-upload.html

```diff
@@ -783,7 +783,6 @@
         }
 
         _onFileRemove(event) {
-          event.stopPropagation();
```

Comment:
Could you add a test for this? Maybe dispatch event manually and check whether it is also fired on the host?

Reply:
Doesn't these https://github.com/vaadin/vaadin-upload/blob/master/test/upload.html#L500 test that the `file-remove` event is dispatched to the `vaadin-upload` element?

Reply:
So it seems 🤔 I will have a closer look.

Reply:
So, the test was there but not failing because we were not calling `stopImmediatePropagation`(which would break the test)  but `stopPropagation` (which prevents propagating the event further but does not prevent other listeners on the same node).